style.css

```diff
@@ -1 +1,148 @@
 /*  Add your styling here */
+body {
+    font-family: 'Roboto', sans-serif;
+    -webkit-font-smoothing: antialiased;
+}
+.headerheader-wrapper {
+    display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+
+    
+}
+.background  {
+    background-image: url("https://c4.wallpaperflare.com/wallpaper/953/450/507/nuts-cream-dessert-cakes-wallpaper-preview.jpg");
+    background-size: cover;
+    background-repeat: no-repeat;
+    height: 582.4px;
+    background-position: center bottom;
+    overflow: auto;
+}
+
+
+.logo {
+    margin-top: 15px;
+    position: relative;
+    display: flex;
+    justify-content: center;
+
+
+  }
+  .logo > img {
+    
+    height: 150px;
+    flex-direction: column;
+    text-align: center;
+    margin-bottom: 20px;
+  }
+  nav > ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-around;
+    font-family: 'Roboto', sans-serif;
+    font-size: 16px;
+    margin-top: 25px;
+    color: #838994;
+    padding-left: 0mm;
+    
+  
+  }
+  li {
+    list-style: none;
+    
+  }
+  li:hover {
+	cursor: pointer;
+	color: #282a85;
+}
+.btn {
+	width: 33.33%;
+	border-radius: 4px;
+    color: #ffffff;
+    background-color: #4794f8;
+    font-size: 16px;
+    display: block;
+    padding: 16px 32px;
+    margin: 0 auto;
+    text-decoration: none;
+    text-align: center;
+}
+
+ .btn:hover {
+	cursor: pointer;
+    background-color: #d9400e;
+    text-decoration: none;
+}
+
+.row {
+    display: flex;
+    flex-wrap: wrap;
+    width: 100%;
+    justify-content: space-around;
+    padding: 0 10px;
+   
+}
+.col-12{
+    width: 100%;
+    padding: 20px 10px;
+    margin-top: 10px;
+
+}
```

Comment:
Don't write your own grid - copy and paste the code from here instead: https://github.com/CodeYourFuture/simple-grid/blob/master/grid.css

And *don't change it*. Remove all your own `col-...` classes.